packages/terra-graphs/src/components/Line/LineGraph.jsx

```diff
@@ -0,0 +1,64 @@
+/* eslint import/no-unresolved: off */
+import React from 'react';
+import PropTypes from 'prop-types';
+import Carbon from '@cerner/carbon-graphs/lib/js/carbon';
+import '../Graph.module.scss';
+import './LineGraph.module.scss';
+import utils from '@cerner/carbon-graphs/lib/js/helpers/utils';
+
+const propTypes = {
+  /**
+   * ID for graph created.
+   */
+  graphID: PropTypes.string.isRequired,
+  /**
+   * Graph configuration object
+   */
+  graphConfig: PropTypes.object.isRequired,
+  /**
+   * data to be displayed in graph
+   */
+  dataset: PropTypes.array,
+  /**
+   * timeout to display multiple data contents in specific time interval.
+   */
+  timeout: PropTypes.array,
+};
+
+const LineGraph = ({
+  graphConfig, dataset, graphID, timeout,
+}) => {
+  React.useEffect(() => {
+    const graph = Carbon.api.graph(graphConfig);
+    if (!(utils.isUndefined(dataset))) {
+      if (!(utils.isUndefined(timeout))) {
+        let count = 0;
```

Comment:
can we use the index within forEach instead of defining a manual count variable?

Reply:
Used index instead of count. Click [here](https://github.com/cerner/terra-graphs/pull/106/commits/8f1b7d01eb1f006f648f922b7b0ca25095b52fb2#diff-5e17603406b642035820d408335769ae48391021a7787fd86404b130590e9606R42) to see the changes.